### DISCUSSION TITLE
plugins  ts destination path is root

### DISCUSSION THREAD

**Discussion by msohailhussain** [adrien2p/medusa-plugins#52] [Q&A] [answered]
Hi, 

Is it possible if the ts destination path can be `./dist` and medusa loads from there. The problem to generate ts to js at root makes harder to track owned vs generated javascript files.

Or may be in medusa, if plugins can resolve all paths with some configurable attribute mentioned in plugins config. e.g. `basedir` or something. 
e.g.
```
${pluginDetails.resolve}/repositories/*.js
```
to 
```
${pluginDetails.resolve}/${pluginDetails.options.basedir}/subscribers/*.js
```

thanks,
Sohail

**Comment by adrien2p** [comment, marked as answer]
I am completely agree with you, this is a subject i have also mentioned internally and something will probably be done towards it. In the meantime, i would suggest you to add that discussion in medusa instead of here as it is link to medusa if that's ok 👍

**Reply by msohailhussain** [reply]
Sure sure. Thanks Adrien. High appreciate.
